plugins/include/sqlx.inc

```diff
@@ -105,171 +129,248 @@ native SQL_QuoteString(Handle:db, buffer[], buflen, const string[]);
+ *                      for when a handle is not available.
+ * @param buffer        Buffer to copy to.
+ * @param buflen        Maximum size of the buffer.
+ * @param fmt           Format of string to backquote (should not overlap buffer).
+ * @param ...           Format arguments.
+ * 
+ * @return              Length of new string, or -1 on failure.
  */
 native SQL_QuoteStringFmt(Handle:db, buffer[], buflen, const fmt[], any:...);
 
 
 #define TQUERY_CONNECT_FAILED	-2
 #define TQUERY_QUERY_FAILED	-1
 #define TQUERY_SUCCESS		0
+
 /**
  * Prepares and executes a threaded query.
- * This will not interrupt gameplay in the event of a poor/lossed 
- *  connection, however, the interface is more complicated and 
- *  asynchronous.  Furthermore, a new connection/disconnection is 
- *  made for each query to simplify driver support.
- * The handler should look like:
+ * @note The handler should look like:
+ *       public QueryHandler(failstate, Handle:query, error[], errnum, data[], size, Float:queuetime)
+ *       failstate - One of the three TQUERY_ defines.
+ *       query     - Handle to the query, do not free it.
+ *       error     - An error message, if any.
+ *       errnum    - An error code, if any.
+ *       data      - Data array you passed in.
+ *       size      - Size of the data array you passed in.
+ *       queuetime - Amount of gametime that passed while the query was resolving.
+ * @note This will not interrupt gameplay in the event of a poor/lossed 
+ *       connection, however, the interface is more complicated and 
+ *       asynchronous. Furthermore, a new connection/disconnection is 
+ *       made for each query to simplify driver support.
+ * @note The handle does no need to be freed.
  *
- * @param failstate - One of the three TQUERY_ defines.
- * @param query - Handle to the query, do not free it.
- * @param error - An error message, if any.
- * @param errnum - An error code, if any.
- * @param data - Data array you passed in.
- * @param size - Size of the data array you passed in.
- * @param queuetime - Amount of gametime that passed while the query was resolving.
- * 
- * public QueryHandler(failstate, Handle:query, error[], errnum, data[], size, Float:queuetime)
- *
- * Note! The handle you pass in is a DB Tuple, NOT an active connection!
- * Note! The handle does not need to be freed.
- * Also note: This function is not guaranteed to be in another thread
- *  (in fact - it's not).  You're seeing data "after the fact", 
- *  and as such to execute another query you should run 
- *  SQL_ThreadQuery again with new data.
+ * @param db_tuple      Tuple handle, returned from SQL_MakeDbTuple().
+ * @param handler       A function to be called when the query finishes. It has to be public.
+ * @param query         The query string.
+ * @param data          Additional data array that will be passed to the handler function.
+ * @param dataSize      The size of the additional data array. 
  */
```

Comment:
Misses ``@noreturn` + `@error` with the following errors `Thread worker was unable to start`, `Invalid info tuple handle`, `Function not found`.